test/debug/main.go

```diff
@@ -53,14 +53,7 @@ func newCmdFromApp(rootCmd *cobra.Command, commands []*plugin.Command) {
 			panic(err)
 		}
 
-		for _, subCmd := range cmd.GetCommands() {
-			subCobraCmd, err := subCmd.ToCobraCommand()
-			if err != nil {
-				panic(err)
-			}
-			cobraCmd.AddCommand(subCobraCmd)
-		}
-
 		rootCmd.AddCommand(cobraCmd)
+		newCmdFromApp(cobraCmd, cmd.GetCommands())
```

Comment:
is this correct? shouldn't this be before adding the `cobraCmd` to the root command?

Reply:
We should add the subcommands. But I'm checking now, and it is not working; the conversion for the Cobra commands only passes the info, not the handler. And the commands show as additional info. We should find a better way to do it

Reply:
created a issue for it